### DISCUSSION TITLE
Datebase error deleting user

### DISCUSSION THREAD

**Discussion by MrVibe** [supabase/supabase#30543] [Questions] [answered]
How do I delete a user from Supabase Authentication.
This fake user has been registered via supabase service role for testing with confirm_email:true and now it can not be removed.

<img width="902" alt="Screenshot 2024-11-19 at 1 00 02 PM" src="https://github.com/user-attachments/assets/1320fda7-0473-4df1-b233-70f74e25d44f">
.

**Comment by GaryAustin1** [comment, marked as answer]
Look in the Postgres logs for the error details.  
Likely you have tables with rows linked thru a foreign key to the user or files in storage linked to the user.   Those have to be deleted first or cascade deleted.
